pulsar-io/hdfs2/src/main/java/org/apache/pulsar/io/hdfs2/AbstractHdfsConnector.java

```diff
@@ -138,12 +137,8 @@ protected void checkHdfsUriForTimeout(Configuration config) throws IOException {
         }
         InetSocketAddress namenode = NetUtils.createSocketAddr(address, port);
         SocketFactory socketFactory = NetUtils.getDefaultSocketFactory(config);
-        Socket socket = null;
-        try {
-            socket = socketFactory.createSocket();
+        try(Socket socket = socketFactory.createSocket()) {
```

Comment:
```suggestion
        try (Socket socket = socketFactory.createSocket()) {
```

Add the space. I think currently the checkstyle plugin is not applied to pulsar-io module.